test/testutil/structs.go

```diff
@@ -62,3 +64,15 @@ func CrdReady(status *apiextensionsv1.CustomResourceDefinitionStatus) bool {
 	}
 	return established && namesAccepted
 }
+
+func NewRukpakOwnerReference(name string, uid types.UID) []metav1.OwnerReference {
+	trueRef := true
+	return []metav1.OwnerReference{{
+		APIVersion:         "core.rukpak.io/v1alpha1",
```

Comment:
nit: is it possible to get this from the api definition? For example I think `rukpakv1alpha1.GroupVersion.String()` should equal this.